components/server-side-render/test/index.js

```diff
@@ -0,0 +1,64 @@
+import httpBuildQuery from 'http-build-query';
+
+// The following tests are adapted to the behavior of http-build-query v0.7.0,
+// to help mitigating possible regressions in the upstream library.
+describe( 'http-build-query', function() {
```

Comment:
Shouldn't we test our own code interacting with this library instead? I don't see any value in having this kind of tests. We use dozens of external libraries and we never test them directly.